src/collection-browser.ts

```diff
@@ -523,6 +538,7 @@ export class CollectionBrowser
         .searchType=${this.searchType}
         .aggregations=${this.aggregations}
         .fullYearsHistogramAggregation=${this.fullYearsHistogramAggregation}
+        .previousSearchType=${this.previousSearchType}
```

Comment:
q: can we reduce the context of `<collection-facets>` by telling it to show its More... links?